test_autoflake.py

```diff
@@ -501,6 +501,41 @@ def foo():
 """
         self.assertEqual(line, ''.join(autoflake.filter_code(line)))
 
+    def test_with_ignore_init_module_imports_flag(self):
+        # Need a temp directory in order to specify file name as __init__.py
+        temp_directory = tempfile.mkdtemp(dir='.')
+        temp_file = os.path.join(temp_directory, '__init__.py')
+        try:
+            with open(temp_file, 'w') as output:
+                output.write('import re\n')
+
+            p = subprocess.Popen(
+                list(AUTOFLAKE_COMMAND) +
```

Comment:
this style (called hanging indent) is compliant with pep8 , but just my personal opinion here, ... I find this style to be not ideal for readability.

when I see a four space indent, I think a block was started.  I know I should also look for a `:` at the end of the previous line, but the brain likes to do less work if it can avoid it.


The alternative is that args should be indented four spaces from the start of the function call on the previous line.

Here the function call is `subprocess.Popen(` -- it starts at `sub..`, so the indentation would be
```py
            p = subprocess.Popen(
                    list(AUTOFLAKE_COMMAND) +
                    ['--ignore-init-module-imports', temp_file],
                    stdout=subprocess.PIPE)
```

That indentation immediately shows there is only one possible meaning for that identation there is no need to look at anything else.

Another style which is increasibly popular is to use the hanging indent, but close it on the same line, which means the eyes focused on the current indentation left, usually near the left, can also see a `)`, and again immediately can recognise what the indented block is, like:

```py
            p = subprocess.Popen(
                list(AUTOFLAKE_COMMAND) +
                ['--ignore-init-module-imports', temp_file],
                stdout=subprocess.PIPE,
            )
```

This codebase doesnt show any preferences.  This comment is just an FYI.  No need to change it, especially for tests and a block this small -- the diffrences in readability is very small, and different people problem have a different readility experience with something like this..

Reply:
Gotcha, will keep in mind for the future, thanks!